appcircle_ios_flutter_build/1.0.3/component.yaml

```diff
@@ -0,0 +1,33 @@
+platform: iOS
+buildPlatform: Flutter
+displayName: "Flutter Build for iOS"
+description: "Build your Flutter project for iOS."
+docsUrl: https://docs.appcircle.io/workflows/flutter-specific-workflow-steps#flutter-install
+requiredComponents: "appcircle_git_clone,appcircle_xcode_select"
+precedingComponents: "appcircle_cocoapods"
```

Comment:
@boztopuz `flutter_install` should be added in `precedingComponents` list

Reply:
It must be added to requiredComp section, because it will install flutter cli. and flutter_build_ios step uses the flutter cli commands. I added to requiredComponents section.